tests/modules/nf-core/fgbio/zipperbams/nextflow.config

```diff
@@ -0,0 +1,11 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: "test_fgbio_zipperbams:FGBIO_ZIPPERBAMS" {
+        ext.args = "--tags-to-reverse Consensus --tags-to-revcomp Consensus"
+        ext.prefix = {"${meta.id}_zipped"}
```

Comment:
```suggestion
```

And then remove it here, because this is not passed along with the module when downloading the module in a pipeline

Reply:
as above. resolved it dynamically from inside the module

Reply:
Even better :partying_face: